.github/workflows/moodle-ci.yml

```diff
@@ -32,8 +32,15 @@ jobs:
       fail-fast: false
       matrix:
         php: ['7.3', '7.4']
-        moodle-branch: ['MOODLE_39_STABLE', 'MOODLE_310_STABLE', 'MOODLE_311_STABLE']
+        moodle-branch: ['MOODLE_310_STABLE', 'MOODLE_311_STABLE']
         database: [pgsql, mariadb]
+        include:
```

Comment:
@pattwell Can this syntax be improved?

Reply:
@Ndibui - We're not doing `moodle-branch: ['MOODLE_310_STABLE', 'MOODLE_311_STABLE', 'MOODLE_400_STABLE']` because we want to run PHP 8.0 only with Moodle 4.0?

Reply:
@kei178 Yes, that's exactly why.